packages/icons/src/utils/css-helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { rem } from '@littlespoon/theme/lib/utils'
+
+/**
+ * Returns rotation value for given direction.
+ * Works only if icon initially points down
+ */
+export function getRotate(direction: 'up' | 'left' | 'right' | 'down' | undefined): number {
```

Comment:
Nit: no need for change but you can do thing with:

```ts
direction?: 'up' | left' | 'right' | 'down'
```

Reply:
Thanks, will update!